### ISSUE TITLE
/tv location

### ISSUE THREAD

**Issue by pettitt87** [whatdaybob/sonarr_youtubedl#3]
Sonarr 2.0.0.5344. Maybe I am just configuring it incorrectly, but I can't figure out how to fix it. My shows are located in a folder /path/to/Episodes.
In sonarr, that path is mapped to /tv. In sonar, the path for everything shows up as /tv/{series}. sonarr_youtubedl is saving things to /path/to/Episodes/tv/{series}. That /tv in there makes it so it's not going to the same directory that sonarr is monitoring

**Comment by DanWillman** [comment]
For what it's worth, I have my plex data at `/media/plex/TV Shows` and when I try to map the volume `/media:/sonarr_root` I noticed it was also including the data folder and downloading to the wrong spot. I fixed this by updating the volume mapping to be `/media:/sonarr_root/data`. 

So perhaps the easier fix is to just update the documentation to reflect that mapping, or map the data folder in your volume after any code changes?
